app/routes/_layout+/assets._index.tsx

```diff
@@ -159,6 +170,27 @@ export async function loader({ context, request }: LoaderFunctionArgs) {
       plural: "assets",
     };
 
+    const headers = new Headers();
```

Comment:
We already have a method of creating and setting headers. Please use the approach we use everywhere. 
Example:
```ts
    const headers = [
      setCookie(
        await setSelectedOrganizationIdCookie(
          organizationsIds.find((orgId) => orgId === qr.organizationId) ||
            personalOrganization.id
        )
      ),
    ];
    ```

Reply:
```const headers = [
      setCookie(userPrefsCookie),
      ...(filtersCookie? [setCookie(filtersCookie)] : [])
    ];

Reply:
> ```
> const headers = [
>       setCookie(userPrefsCookie),
>       ...(filtersCookie? [setCookie(filtersCookie)] : [])
>     ];
> ```

this what you mean